.env.example

```diff
@@ -38,12 +38,14 @@ CLIENT_ID_HEROKU=
 CLIENT_ID_VERCEL=
 CLIENT_ID_NETLIFY=
 CLIENT_ID_GITHUB=
+CLIENT_ID_GITHUB_ENVIRONMENT=
 CLIENT_ID_GITLAB=
 CLIENT_ID_BITBUCKET=
 CLIENT_SECRET_HEROKU=
 CLIENT_SECRET_VERCEL=
 CLIENT_SECRET_NETLIFY=
 CLIENT_SECRET_GITHUB=
+CLIENT_SECRET_GITHUB_ENVIRONMENT=
```

Comment:
My initial thought was we can assign different permissions for the oauth keys for both the integrations, but since anyways we will be merging both GitHub and GitHub environments, we can simply reuse the GitHub client id and secrets.